tests/integrations/raftstore/test_conf_change.rs

```diff
@@ -929,3 +929,40 @@ fn test_conf_change_fast() {
     must_get_equal(&cluster.get_engine(2), b"k1", b"v1");
     assert!(timer.saturating_elapsed() < Duration::from_secs(5));
 }
+
+#[test]
+fn test_crash_when_remove_node_from_two_member_cluster() {
+    let mut cluster = new_node_cluster(0, 2);
+    let pd_client = Arc::clone(&cluster.pd_client);
+    pd_client.disable_default_operator();
+
+    let r1 = cluster.run_conf_change();
+    pd_client.must_add_peer(r1, new_peer(1, 1));
```

Comment:
run_conf_change will add the first peer automatically.

Reply:
Thanks for your reply, I have address the comments in the above.